src/server_manager/web_app/app.ts

```diff
@@ -187,11 +176,10 @@ export class App {
     });
 
     appRoot.addEventListener('OpenImageRequested', (event: PolymerEvent) => {
-      sendElectronEvent('open-image', event.detail.imagePath, event.detail.windowTitle);
+      openImage(event.detail.imagePath);
```

Comment:
This file could also benefit from an explicit referende to the .d.ts file: https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html